docs/feature-comparison.md

```diff
@@ -0,0 +1,59 @@
+# Feature Comparison
+
+The following is an incomplete list of features differences between the legacy project system and the new project system. 
+
+**Feature**|**Legacy**|**New**|**Notes**
+---|---|---|---
+**Platforms**                                                               |
+.NET Standard                                                               | No       | Yes
```

Comment:
.NET Standard is supported by csproj (at least through 1.6) ([example](https://github.com/sharwell/antlr4cs/blob/942fb4b7dfd53b78db523234c5526bf432c95cee/runtime/CSharp/Antlr4.Runtime/Antlr4.Runtime.netstandard.csproj))

Reply:
These are not standard usages, and the feature has been turned off in newer versions.